Js/index.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable consistent-return */
+const myForm = document.querySelector('form');
+const ply1 = document.getElementById('pl1');
+const ply2 = document.getElementById('pl2');
+const contentField = document.querySelector('.content');
+const input1 = document.getElementById('p1');
+const input2 = document.getElementById('p2');
+const replay = document.getElementById('replay');
+const reload = document.getElementById('reload');
+const gameBoard = document.getElementById('board');
+const winner = document.getElementById('winner');
+
+const moves = ['', '', '', '', '', '', '', '', ''];
+const winningCases = [
+  [0, 1, 2],
+  [0, 4, 8],
+  [0, 3, 6],
+  [1, 4, 7],
+  [2, 5, 8],
+  [2, 4, 6],
+  [3, 4, 5],
+  [6, 7, 8],
+];
+
+function Players() {
+  // eslint-disable-next-line no-unused-expressions
+  this.p1 = 'X';
+  this.p2 = 'O';
+}
+
+const players = new Players();
+let won = false;
+
+function changePlayer(p) {
+  const { p1, p2 } = players;
+  return p % 2 === 0 ? [p1, p2] : [p2, p1];
+}
+
+function playerSign(p) {
+  return p === players.p1 ? 'X' : 'O';
+}
+
+function playerMoves(s) {
+  const arr = [];
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < moves.length; i++) {
+    if (moves[i] === s) {
+      arr.push(i);
+    }
+  }
+  this.steps = arr;
+  return arr;
+}
+function checkWin(b) {
+  const w = winningCases;
+  for (let i = 0; i < w.length; i++) {
+    if (b.includes(w[i][0]) && b.includes(w[i][1]) && b.includes(w[i][2])) {
+      return true;
+    }
+  }
+}
+
```

Comment:
According to the instructions from `Odin` you are supposed to have the following classes `game`, `player`, and `gameboard`, don't forget to make use of factory functions.
![image](https://user-images.githubusercontent.com/31897434/88722986-04ecae00-d120-11ea-8b56-202c4ff7f05e.png)
https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript